### PR TITLE
AH Develop

### PR DESCRIPTION
**Pull request by hortonau**
This has the helper functions/logic for modifying flags.

### CODE REVIEW

File: emulator.h [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-#define MEM_SIZE 65536
+// Flags Defined
+#define FLAG_S 0x80  // NOLINT
+#define FLAG_Z 0x40  // NOLINT
+#define FLAG_AC 0x20 // NOLINT
+#define FLAG_P 0x10  // NOLINT
+#define FLAG_CY 0x08 // NOLINT
+
+// Memory
+#define MEM_SIZE 65535 // NOLINT
```
Comment (cliff-bielinski) [inline emulator.h]: Should be 65536?

**Review by cliff-bielinski — CHANGES_REQUESTED** [review]
Remove testoutput from files to be merged. 

Change MEMSIZE back to 65536.

File: shell.c [hunk 2]
```diff
@@ -18,6 +18,10 @@ main(int argc, char *argv[])
   // initialize CPU state
   i8080 cpu;
   cpu_init(&cpu);
+
+  const char *rom_path = "./invaders";
+  // NOLINTNEXTLINE
+
```
Comment (cliff-bielinski) [inline shell.c]: Should be deleted now that the filepath is passed as an argument.

Comment (Roamingturtl) [inline shell.c]: Cliff changed this to not be hardcoded and to take arguments instead.

File: shell.c [hunk 3]
```diff
@@ -29,6 +33,10 @@ main(int argc, char *argv[])
 
   while (true)
     {
+
+      // 1 Fetch, decode, and execute next instruction
```
Comment (cliff-bielinski) [inline shell.c:35]: This comment was deleted from earlier merge. Can be removed again

File: emulator.h (hunk 1 above)
Comment (hortonau) [reply emulator.h]: updated

**Review by Roamingturtl — APPROVED** [review]
changes requested all completed.

**Review by cliff-bielinski — APPROVED** [review]

**Pull request merged**